docs/pages/server-access/guides/openssh.mdx

```diff
@@ -23,13 +29,36 @@ We've outlined these reasons in [OpenSSH vs Teleport SSH for Servers?](https://g
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-- A Linux host with the OpenSSH server `sshd` installed, but not Teleport. The
-  SSH port on this host must be open to traffic from the Teleport Proxy Service
-  host.
+- A Linux host with the OpenSSH server `sshd` version 7.4 or above installed,
+  but not Teleport. The SSH port on this host must be open to traffic from the
+  Teleport Proxy Service host.
 
 (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/4. Configure `sshd` to trust the Teleport CA
+## Step 1/5. Add a node resource to your Teleport cluster
+
+Teleport needs to know how to reach your OpenSSH server, so a node resource must
+be created. On your local machine, create a file `openssh-node-resource.yaml`:
+
+```yaml
+kind: node
+sub_kind: openssh
+metadata:
+  labels:
+    env: prod
+spec:
+  addr: 1.2.3.4:22
```

Comment:
Will this `addr` apply to the environments of all users following this guide? Ideally, it should be possible for a user to go through the steps themselves and achieve a working result.

Reply:
This document already used example values for a few things so that's what I did too. Would you prefer I use placeholders instead such as `<ADDR>`?